vhdbuilder/packer/components.json

```diff
@@ -239,12 +239,16 @@
       ],
       "multiArchVersions": [
         "v0.7.11",
+        "v0.7.12",
         "v0.7.9",
         "v1.0.6",
         "v1.0.8",
+        "v1.0.9",
         "v1.1.2",
         "v1.1.4",
-        "v1.23.1"
+        "v1.1.5",
+        "v1.23.1",
+        "v1.23.2"
       ]
```

Comment:
could we remove `v0.7.9` / `v1.0.6` / `v1.1.2`? @feiskyer

Reply:
yes, please remove the oldest patch version

Reply:
done